features/bevragen/autorisatie.feature

```diff
@@ -0,0 +1,457 @@
+      | type     | https://learn.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=net-7.0#system-net-httpstatuscode-forbidden |
+      | title    | U bent niet geautoriseerd voor de gebruikte parameter(s).                                                               |
+      | status   | 403                                                                                                                     |
+      | detail   | U bent niet geautoriseerd voor het gebruik van parameter(s): geslacht.                                                  |
+      | code     | unauthorizedParameter                                                                                                   |
+      | instance | /haalcentraal/api/brp/personen                                                                                          |
+
+
+  Rule: Wanneer met fields gevraagd wordt om een veld waarvoor de gebruiker niet geautoriseerd is, wordt een foutmelding gegeven
+    Om een veld te mogen vragen moet de afnemer geautoriseerd zijn voor alle LO BRP rubrieken waar het veld mee gevuld of van afgeleid wordt
+
+    @fout-case
+    Scenario: Afnemer vraagt om een veld waarvoor deze niet geautoriseerd is
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)     | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 10210 10220 10230 10240 10310 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode |        |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+      | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | 19830526              | Maassen               | Pieter            | M                           |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                            |
+      | type                | RaadpleegMetBurgerservicenummer                                   |
+      | burgerservicenummer | 000000024                                                         |
+      | fields              | burgerservicenummer,geboorte.datum.geboorte.plaats,naam.voornamen |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | detail   | De foutieve parameter(s) zijn: fields[2].                               |
```

Comment:
Is 'De foutieve field waarden zijn:' niet duidelijker?

Reply:
aangepast